src/App.tsx

```diff
@@ -128,6 +131,33 @@ class App extends Component<void, AppState> {
     });
   }
 
+  pauseCurrentPlayback(): void {
```

Comment:
Could probably combine these 2 methods into 1 like:

```
adjustPlayback(paused: boolean): void {
   this.setState({paused})
    (paused ? this.spotifyService.pauseCurrentPlayback() : this.spotifyService.resumeCurrentPlayback()).then(res => {
      res.body?.getReader().read().then(body => {
        if(body?.done) {
          setTimeout(() => this.fetchCurrentlyPlaying(), 1000);
        }
      });
    });
  }
```

Reply:
Could even make it not have any parameters and just invert what's currently the state of `paused` 🤔

Reply:
You thinking something like?

```this.setState({ paused: !paused})```

Reply:
Yeah could go with 
```
invertPlayback(): void {
   
    (!this.state.paused ? this.spotifyService.pauseCurrentPlayback() : this.spotifyService.resumeCurrentPlayback()).then(res => {
      res.body?.getReader().read().then(body => {
        if(body?.done) {
          setTimeout(() => this.fetchCurrentlyPlaying(), 1000);
        }
      });
    });
    this.setState({pause: !this.state.paused})
  }
  ```
  
 Though doesn't really matter, up to you my man